### DISCUSSION TITLE
Option to show Taskbar on top of the screen

### DISCUSSION THREAD

**Discussion by ROOBROMA** [lawand-dot-io/taskbar#48] [Feature requests]
Hi,

I love the taskbar, great job and Thanks.
I would love to see the option to relocate the task bar to the top of the screen instead below.

Many thanks!

**Comment by lawand-dot-io** [comment]
Additional feedback from one user: "as a multi-tasker with a bunch of different windows/tabs open. If everything were at the top then I'd have much quicker access to the menubar, taskbar, and application headers. Typically, menus, tabs, and other "actionable" items are all at the top of the screen. I used this feature a ton in Windows and made it so much faster to manage applications etc. "

**Reply by jimicze** [reply]
I am using taskbar as well in Linux, Windows 10/11, so i am missing this feature a lot here.

**Comment by elstevo711** [comment]
Wanted to add to this feature request as well. I have a 65" TV as a monitor and currently use uBar at the top of my screen. This works well for me as I have my "monitor" at desk level (the bottommost bezel is sitting on my desktop), so having anything sitting  on my desk makes it hard to see the taskbar items if it is on the bottom. I also use a top uBar on my 16" MBP. I also use top panels in Linux (Fedora 44 KDE) on 1 desktop and 1 laptop. I find the top bar is more useful and accessible to my workflow.

I am loving Taskbar, and as per your admission, it is active and updating often, whereas I have had to wait almost a year for uBar to update before upgrading to the latest macOS to make sure of compatibility with my workflow. (Made that mistake before and had to downgrade back to former OS version just for compatibility.)

Just having the option would be phenomenal. This is my main reason for sticking with uBar currently. Which I will add I would love to ditch, as their development is slow and their customer service sucks!

**Reply by lawand-dot-io** [reply]
Thank you for the additional input 👍
